packages/tc-ui/src/primitives/enum/__tests__/e2e-enum-primitive.cyp.js

```diff
@@ -0,0 +1,50 @@
+const { executeQuery, dropFixtures } = require('../../../test-helpers/db');
+
+const namespace = 'e2e-primitives-enum';
+const code = `${namespace}-code`;
+
+const save = () =>
+	cy.get('[data-button-type="submit"]').click({
+		force: true,
+	});
+
+const createRecord = (props = {}) =>
+	cy.wrap(
+		executeQuery(`CREATE (:PropertiesTest $props)`, {
+			props: { code, ...props },
+		}),
+	);
+
+describe('End-to-end - Enum primitive', () => {
```

Comment:
Highlight #14 These specs for primitive components now follow a quite standard pattern
- view empty state (have uncovered that some components are buggy in this respect, so have raised issues and skipped tests for now)
- edit empty state
- add a value
- edit a value
- remove a value
- other things which will vary per component

The test suite fills me with more confidence now